TeleSharp.Generator/Program.cs

```diff
@@ -429,4 +631,18 @@ public static FileStream MakeFile(string path)
         }
     }
 
+    struct Property
+    {
+        public string type;
```

Comment:
Can you use PascalCase (instead of lowercase/camelCase) for public members, to follow .NET coding guidelines?

Also better add a constructor that receives both type and name values as parameters of it.